src/components/navigation/_navigation.scss

```diff
@@ -6,14 +6,18 @@
         margin-top: -0.03rem;
 
         &--neutral {
-            background: var(--ons-color-header-neutral);
+            background: var(--ons-color-grey-75);
```

Comment:
As this is the background for the header we should be using one of the named colours here like `--ons-color-header-neutral` and maybe we will need to update what it is set to in colors.scss. But we will need to take into count any other places it may be used. Same with the navigation link below